csrc/scheduler/matmul_utils.cpp

```diff
@@ -307,32 +309,63 @@ std::string getMatmulCompileTimeRejectReason(Fusion* fusion) {
   // 1. check if there is exactly one MmaOp defined in the fusion
   // 2. check if MmaOp inputs match any of supported inputs layout
   // 3. check if fusion represents expressions that are recognized by matmul
-  //    scheduler
+  // scheduler In addition to the above, it may be the case that there is no mma
```

Comment:
As an incremental step, it might makes sense to allow preexisting `MmaOp`s. But after things are built up, I prefer to make the pass you added the only source that creates `MmaOp`s, so at that time, we should have "it will always be the case that" instead of "it may be the case that"

Reply:
Actually, it seems it's always the case? This function is called from `getMatmulCompileTimeRejectReason`, which fills a proper `props` value even when an existing `MmaOp` is used. 

https://github.com/NVIDIA/Fuser/pull/1507/files#diff-b979e55bd012809d32c2fe0641c25ba3c15b8b447deea459d80a13bfb0a1fadaR361

So, it seems we should always just use the given `props` parameter.